akd_core/Cargo.toml

```diff
@@ -33,8 +33,8 @@ parallel_vrf = ["tokio"]
 
 bench = ["parallel_vrf", "blake3", "vrf", "tokio/rt-multi-thread"]
 
-# Default features mix (no hash function is selected at this point)
-default = ["vrf"]
+# Default features mix (with blake3 as the default hash function)
+default = ["vrf", "blake3"]
```

Comment:
Hmm we used to have this, and i remember it breaking something (like running `akd` without blake3 hashing or something). But the CI is happy so clearly i'm wrong....

But perhaps some other updates fixed it. I think it had to do with like `akd` pulling in a different mix of functionality in the `dev` profile, but that would break the ci